v1/message_test.go

```diff
@@ -10,13 +10,13 @@ import (
 )
 
 var (
-	testMessageBytes  = []byte(`["~#c4",["abc123","text/plain","~:public-group-user-message",154593077368201,1545930773682,["^ ","~:chat-id","testing-adamb","~:name", "test-name","~:response-to", "id","~:text","abc123"]]]`)
+	testMessageBytes  = []byte(`["~#c4",["abc123",1,1,154593077368201,1545930773682,["^ ","~:chat-id","testing-adamb","~:name", "test-name","~:response-to", "id","~:text","abc123"]]]`)
 	testMessageStruct = Message{
-		Text:      "abc123",
-		ContentT:  "text/plain",
-		MessageT:  "public-group-user-message",
-		Clock:     154593077368201,
-		Timestamp: 1545930773682,
+		Text:        "abc123",
+		ContentType: 1,
+		MessageType: 1,
+		Clock:       154593077368201,
+		Timestamp:   1545930773682,
 		Content: Content{
 			ChatID:     "testing-adamb",
 			Text:       "abc123",
```

Comment:
why is text still kept in two places?

Reply:
that's still the old schema (transit), still not sure if I will drop support for it in this PR or keep compatibility but publish on protobuf (less changes)